wiki/Game_mode/en.md

```diff
@@ -1,42 +1,50 @@
 
 ![Mode selection interface](/wiki/shared/Interface_mode.png "Mode selection interface")
 
 ## ![][osu!] osu!
 
 *Main page: [osu!](osu!)*
 
-*osu!* is the default game mode in the osu! client, and is based on the original [iNiS'](https://en.wikipedia.org/wiki/INiS "Wikipedia") rhythm video game *[Osu! Tatakae! Ouendan](https://en.wikipedia.org/wiki/Osu!_Tatakae!_Ouendan "Wikipedia")*. In osu!, the player is expected to tap hit circles, hold sliders, and spin spinners to the beat of songs.
+*osu!* is the default mode, where players are expected to tap hit circles, hold sliders, and revolve spinners to the beat of songs. It is based on the original [iNiS'](https://en.wikipedia.org/wiki/INiS "Wikipedia") rhythm video game *[Osu! Tatakae! Ouendan](https://en.wikipedia.org/wiki/Osu!_Tatakae!_Ouendan "Wikipedia")*. osu!'s mascot is [pippi](/wiki/Mascots#-pippi), a pink/black-haired girl.
 
 ![Gameplay of osu!](/wiki/shared/osu-gameplay.jpg "osu! gameplay")
 
 ## ![][osu!taiko] osu!taiko
 
 *Main page: [osu!taiko](osu!taiko)*
 
-*osu!taiko* is based on the *[Taiko no Tatsujin](https://en.wikipedia.org/wiki/Taiko_no_Tatsujin "Wikipedia")* game series produced by [Bandai Namco Entertainment](https://en.wikipedia.org/wiki/Bandai_Namco_Entertainment "Wikipedia"). Made to mimic the feel of playing a real drum, the player makes use of a keyboard or drum kit to hit differently-colored circles on a single conveyor.
+*osu!taiko* simulates the process of playing a real drum, and players make use of a keyboard or drum kit to hit differently-colored circles on a single conveyor. It is based on the *[Taiko no Tatsujin](https://en.wikipedia.org/wiki/Taiko_no_Tatsujin "Wikipedia")* game series produced by [Bandai Namco Entertainment](https://en.wikipedia.org/wiki/Bandai_Namco_Entertainment "Wikipedia"). osu!taiko's mascot is [Mocha](/wiki/Mascots#-mocha), a green-haired girl in a kimono holding drumsticks.
 
 ![Gameplay of osu!taiko](/wiki/shared/taiko-gameplay.jpg "osu!taiko gameplay")
 
 ## ![][osu!catch] osu!catch
 
 *Main page: [osu!catch](osu!catch)*
 
-*osu!catch* is loosely based on a minigame inside of a [Beatmania](https://en.wikipedia.org/wiki/Beatmania "Wikipedia") simulator, which itself is believed to be based on the *EZ2CATCH* mode from the [EZ2DJ](https://en.wikipedia.org/wiki/EZ2DJ "Wikipedia") series. The game mode consists of falling fruit of differing varieties and a "catcher" holding a plate to catch the fruit.
+In *osu!catch*, fruits of differing varieties fall to the beat from the top of the screen. The player controls a [catcher](/wiki/Mascots#-yuzu) with a plate atop their head for catching fruits. This mode is loosely based on a minigame inside of a [Beatmania](https://en.wikipedia.org/wiki/Beatmania "Wikipedia") simulator, which itself is believed to be based on the *EZ2CATCH* mode from the [EZ2DJ](https://en.wikipedia.org/wiki/EZ2DJ "Wikipedia") series. osu!catch's mascot is [Yuzu](/wiki/Mascots#-yuzu), a blue-haired boy.
```

Comment:
```suggestion
In *osu!catch*, fruits of differing varieties fall from the top of the screen, and the player must catch them to the beat. To this end they control a [catcher](/wiki/Mascots#-yuzu) with a plate atop their head for catching said fruits. This mode is loosely based on a minigame inside of a [Beatmania](https://en.wikipedia.org/wiki/Beatmania "Wikipedia") simulator, which itself is believed to be based on the *EZ2CATCH* mode from the [EZ2DJ](https://en.wikipedia.org/wiki/EZ2DJ "Wikipedia") series. osu!catch's mascot is [Yuzu](/wiki/Mascots#-yuzu), a blue-haired boy.
```

Reply:
uhh kind of wanted to reduce the amount of `ca(t-)ching`s here, any other suggestion? I'm fine with the rest of the suggestion